students/M_Sunday/lesson03/Mailroom_Part_1.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+from operator import itemgetter
+
+# Lesson 03, Mailroom Part 1 Assignment
+
+
+# ------- Global Assignments -------
+donor_data = [['John', 'Marks', 5000.34, 2500, 1267],
+              ['Matt', 'Smith', 750],
+              ['Kelsey', 'Rodgers', 55.76],
+              ['John', 'Williams', 500, 1898],
+              ['Morgan', 'Stanley', 12000.98, 8670],
+              ['Roger', 'Johnson', 264, 942.23],
+              ['Sherry', 'Wilson', 4000, 2324.1, 1352],
+              ['Alex', 'Grant', 8739, 4312]]
+
+# ----------------------------------
+
+
+def menu_disp():
+    print("\n\n----- MENU -----")
+    print("1: Send a Thank You")
+    print("2: Create a Report")
+    print("3: Quit")
+    menu_sel_ = input("------> "
+                      "Select an option from the menu: ")
+    return menu_sel_
+
+
+def list_display(data):
+    print("\n---- List of all Donors ----")
+    for item in data:
+        print("- {} {}".format(item[0], item[1]))
+    print("----------------------------")
+    thank_you()
+
+
+def list_details(data):
+    header = "\n{:<18}| {:<12}| {:<10}| {:<12}".format("Donor Name", "Total "
+                                                       "Given", "Num Gifts",
+                                                       "Average Gift")
+    buffer = "------------------------------------------------------------------"
+    print(header)
+    print(buffer)
+    for item in sorted(data, key=itemgetter(-2), reverse=True):
+        print("{:<18} $  {:>9.2f}  {:>10}  $   {:>9.2f}".format(item[0] + ' ' + item[1],
+                                                                item[-2], item[-3],
+                                                                item[-1]))
+
+
+def compose_email(donor_name, donation_amount):
+    greetingemail = "\nDear {},".format(donor_name)
```

Comment:
Python convention is to use `snake_case` for variable and function names (words separated by underscores)